src/qibolab/platforms/abstract.py

```diff
@@ -67,6 +67,7 @@ class Qubit:
     name: str
     readout: Channel
     feedback: Channel
+    twpa: Optional[Channel] = None
```

Comment:
That's fine but can you write somewhere (in the docstring or here) what is the TWPA, because at least to me it is not as clear as readout, drive, etc.

Reply:
Btw, you can document class attributes with individual docstrings:
```py
class A:
    a = None
    """My attribute description"""
```
https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoattribute

In practice, it is working with the basic setup, the link is just the reference to show you it is available.